src/main/java/com/bootcamp/demo/models/User.java

```diff
@@ -0,0 +1,72 @@
+package com.bootcamp.demo.models;
+
+import java.util.Objects;
+
+public final class User {
+    private final Long userId;
+    private String email;
+    private String firstName;
+    private String lastName;
+
+    public User(Long userId, String email, String firstName, String lastName) {
+        this.userId = userId;
+        this.email = email;
```

Comment:
The user information needs to be non null. For ex. the user email, firstname, lastname...etc. cant be null. Check the values here.